src/core/Akka.TestKit/TestKitBase.cs

```diff
@@ -710,10 +718,41 @@ public IActorRef CreateTestActor(string name)
             return CreateTestActor(_testState.System, name);
         }
 
+        private IActorRef CreateInitialTestActor(ActorSystem system, string name)
+        {
+            var testActorProps = Props.Create(() => new InternalTestActor(_testState.Queue))
+                .WithDispatcher("akka.test.test-actor.dispatcher");
+            
+            // IMPORTANT: For the initial TestActor, we need special handling to avoid deadlocks
+            // when running tests in parallel. The default SystemActorOf uses async initialization
+            // which can deadlock when the TestActor runs on CallingThreadDispatcher.
+            //
+            // The issue is that when async=true, a RepointableActorRef is created and its
+            // Initialize(async) method sends a Supervise message but doesn't immediately call Point().
+            // This Supervise message needs to be processed on the CallingThreadDispatcher thread,
+            // but that thread may be blocked waiting for initialization to complete.
+            var systemImpl = system.AsInstanceOf<ActorSystemImpl>();
+            var guardian = systemImpl.Provider.SystemGuardian;
+            var path = guardian.Path / name;
+    
+            var dispatcher = systemImpl.Dispatchers.Lookup("akka.test.test-actor.dispatcher");
+            var mailboxType = systemImpl.Mailboxes.GetMailboxType(testActorProps, dispatcher.Configurator.Config);
+    
+            var repointableRef = new RepointableActorRef(
+                systemImpl, testActorProps, dispatcher, mailboxType, guardian, path);
+    
+            // Initialize synchronously by passing async=false
+            repointableRef.Initialize(async: false);
```

Comment:
this is the key fix - forcing synchronous initialization of the `TestActor` so the `CallingThreadDispatcher` doesn't have to get poked to do anything while we're waiting for the actor to start up.